src/containers/transfer/components/EthTransferForm.js

```diff
@@ -0,0 +1,164 @@
+import React from "react";
+import { connect } from "react-redux";
+import { reduxForm, SubmissionError, Field } from "redux-form";
+import store from "modules/store";
+import { EthSubmissionErrorPanel, EthSubmissionSuccessPanel } from "components/MsgPanels";
+import Button from "components/augmint-ui/button";
+import { Form, Validations, Normalizations } from "components/BaseComponents";
+import { transferEth, ETH_TRANSFER_SUCCESS } from "modules/reducers/ethTransfer";
+import theme from "styles/theme";
+
+class EthTransferForm extends React.Component {
+    constructor(props) {
+        super(props);
+        this.initialVal = this.props.initialValues.ethAmount;
+        this.state = {
+            result: null,
+            amount: this.initialVal
+        };
+        this.handleSubmit = this.handleSubmit.bind(this);
+        this.onEthAmountChange = this.onEthAmountChange.bind(this);
+        this.toggleEthTransferForm = this.toggleEthTransferForm.bind(this);
+    }
+
+    toggleEthTransferForm(e) {
+        this.props.toggleEthTransferForm(e);
+    }
+
+    onEthAmountChange(e) {
+        let amount;
+        try {
+            amount = parseFloat(e.target.value);
+            if (isNaN(amount)) {
+                amount = 0;
+            } else {
+                amount = parseFloat(e.target.value);
+            }
+        } catch (error) {
+            return;
+        }
+        this.setState({ amount });
+    }
+
+    async handleSubmit(values) {
+        const ethAmount = parseFloat(values.ethAmount);
+        const res = await store.dispatch(
+            transferEth({
+                payee: this.props.address,
+                ethAmount
+            })
+        );
+        if (res.type !== ETH_TRANSFER_SUCCESS) {
+            throw new SubmissionError({
+                _error: res.error
+            });
+        } else {
+            this.setState({
+                result: res.result,
+                to: this.props.address,
+                ethAmount
+            });
+        }
+    }
+
+    render() {
+        const {
+            error,
+            handleSubmit,
+            submitting,
+            submitSucceeded,
+            clearSubmitErrors,
+            augmintToken,
+            isFunctional,
+            submitText
+        } = this.props;
+
+        const amountInEur = (this.props.rates.info.ethFiatRate * this.state.amount).toFixed(2);
+        const coverdTxs = Math.round((this.state.amount / this.initialVal) * 5);
+
+        return (
+            <div style={isFunctional && { display: "inline" }}>
+                {submitSucceeded && (
+                    <EthSubmissionSuccessPanel
+                        header="ETH transfer submitted"
+                        result={this.state.result}
+                        onDismiss={() => this.toggleEthTransferForm(false)}
+                    >
+                        <p>
+                            Transfer {this.state.ethAmount} ETH to {this.state.to}
```

Comment:
Use `<ETH amount={…} />` component to display ETH amounts and drop the `.toFixed` call above.

Reply:
don't forget to remove `toFixed(2)`

Reply:
@phraktle  it seems i did not forget it, it is used for EUR on line 76